integration-deployment.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
```

Comment:
This file shouldn't needed as we'll use concourse CF support to deploy

Reply:
yep, will now remove.. we were testing before concourse was there